### ISSUE TITLE
Face enhancement erroring result

### ISSUE THREAD

**Issue by xh4010** [microsoft/Bringing-Old-Photos-Back-to-Life#261]
I ran the python GUI. py command, and there was no error, but the generated face image was wrong
```
Running Stage 1: Overall restoration
detection.py:25: DeprecationWarning: BICUBIC is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BICUBIC instead.
  def data_transforms(img, full_size, method=Image.BICUBIC):
initializing the dataloader
model weights loaded
directory of testing image: G:\DeepLearning\pytorch\Bringing-Old-Photos-Back-to-Life-master\test_images\old_w_scratch
processing a.png
processing b.png
processing c.png
processing d.png
test.py:18: DeprecationWarning: BILINEAR is deprecated and will be removed in Pillow 10 (2023-07-01). Use Resampling.BILINEAR instead.
  def data_transforms(img, method=Image.BILINEAR, scale=False):
You are using NL + Res
Now you are processing a.png
Now you are processing b.png
Now you are processing c.png
Now you are processing d.png
Finish Stage 1 ...


Running Stage 2: Face Detection
1
1
1
2
Finish Stage 2 ...


Running Stage 3: Face Enhancement
The main GPU is
0
dataset [FaceTestDataset] of size 5 was created
The size of the latent vector size is [8,8]
Network [SPADEGenerator] was created. Total number of parameters: 92.1 million. To see the architecture, do print(network).
hi :)
Finish Stage 3 ...


Running Stage 4: Blending
Finish Stage 4 ...


All the processing is done. Please check the results.
```

![](https://raw.githubusercontent.com/xh4010/imageBed/master/2023/20230314091638.png)

![](https://raw.githubusercontent.com/xh4010/imageBed/master/2023/20230314091456.png)

**Comment by zhaop-l** [comment]
I have encountered the same problem as you. How did you solve it?
![image](https://github.com/microsoft/Bringing-Old-Photos-Back-to-Life/assets/50993924/2e1c99bd-e603-4f4c-8c12-ebbf83f2a9d8)

**Comment by m45t3rc0d3r** [comment]
I have the same problem. Anybody has a solution for this?

**Comment by Haoming02** [comment]
I stumbled upon this problem as well while developing this [Extension](https://github.com/Haoming02/sd-webui-old-photo-restoration)

For me, the issue lies in the fact that the default path for the `Face Enhancement` checkpoints is set to `./checkpoints`. And since I modified and ran the scripts from elsewhere, the path simply pointed to nothing. The funny thing is that, finding no checkpoints does **not** raise any error at all. That's why it's simply generating noises.

**TL;DR:** Adding `--checkpoints_dir <path to checkpoints folder>` while calling `test_face.py` solved this issue for me.
